.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+
+jobs:
+  audit_and_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.2.0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 14
```

Comment:
I think you need to use `node-version: 16` here and below since it's what we are using locally.

Reply:
Node version 14.18 was used in circleci!
I would also suggest version 16!